transport/transport_test.go

```diff
@@ -53,12 +55,38 @@ func TestTransportForwordAndDial(t *testing.T) {
 }
 
 func TestTransportDialTLS(t *testing.T) {
```

Comment:
https://github.com/haxii/fastproxy/blob/develop/mitm/mitm_test.go

read this code and refactor this ugly shit